src/main/features/core/tray.js

```diff
@@ -31,21 +36,37 @@ let audioDeviceMenu = [
   },
 ];
 
-appIcon = new Tray(path.resolve(trayNormalPath, appIconFileName));
+setAppIconFileName();
+
+appIcon = new Tray(appIconFileName);
+
+Emitter.on('settings:set', (event, details) => {
```

Comment:
This should be changed to

`Settings.onChange('appIconInvert', (newValue) => {`